sonobuoy-plugin.yaml

```diff
@@ -0,0 +1,11 @@
+sonobuoy-config:
+  driver: Job
+  plugin-name: k8s-service-validator-sonobuoy-plugin
+  result-format: raw
+spec:
+  image: projects-stg.registry.vmware.com/tkgdev/yzac/k8s-svc-validator:latest
```

Comment:
are we  allowed to use the vmware registry here?

Reply:
changed to use dockerhub